integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -54,7 +54,15 @@ def del_keys(key_list, c_val, g_val):
 
 def fixup_operation_metrics(opm):
     """Update the specified operationMetrics node to facilitate log comparisons"""
-    for k in "executionTimeMs", "numOutputBytes", "rewriteTimeMs", "scanTimeMs":
+    # note that we remove many byte metrics because number of bytes can vary
+    # between CPU and GPU.
+    # we remove deletion vector metrics because we do not support those yet
+    # see https://github.com/NVIDIA/spark-rapids/issues/8554
+    metrics_to_remove = ["executionTimeMs", "numOutputBytes", "rewriteTimeMs", "scanTimeMs",
+                         "numRemovedBytes", "numAddedBytes", "numTargetBytesAdded", "numTargetBytesInserted",
+                         "numTargetBytesUpdated", "numTargetBytesRemoved",
+                         "numDeletionVectorsAdded", "numDeletionVectorsRemoved"]
```

Comment:
It is not ideal to just ignore the deletion vector metrics. I am still trying to find a better solution.

Reply:
as @jlowe noted in [TBD](https://github.com/NVIDIA/spark-rapids/pull/8628#discussion_r1246765644):

"Seems like there's two phases to supporting deletion vectors. There's full support which is tracked by https://github.com/NVIDIA/spark-rapids/issues/8554, but before that is completed we should be reporting the metrics (always 0) in the delta log stats just like the CPU does when not using deletion vectors. I could see shipping without full deletion support, but we should be adding these deletion metrics even when we don't support them being non-zero. I'm OK with that being a followup, but IMO having the metric is a P0 for the release while the full support issue is not."